README.md

```diff
@@ -23,7 +23,7 @@ Canadarm can be installed in the following ways:
 
 * Reference a hosted version provided by [jsdelivr][jsdelivr]:
```

Comment:
@LukasDrgon can you fix the link here and set it to https://www.jsdelivr.com/package/npm/canadarm ?